code/modules/clothing/head/helmet.dm

```diff
@@ -498,10 +509,13 @@
 	icon_state = "inteq_helmet"
 	can_flashlight = TRUE
 
-/obj/item/clothing/head/helmet/alt/minutemen
+/obj/item/clothing/head/helmet/minutemen
```

Comment:
subtype it under the bulletproof helmet if you're going to use bulletproof helmet stats